### DISCUSSION TITLE
Failed to retrieve token from superset

### DISCUSSION THREAD

**Discussion by peanut00** [apache/superset#26277] [Q&A / Help]
Hello,

I'm trying to get a tocken from superset, but I receiving a status code of 500 with a message {"message":"Fatal error"}

Could you help please?

import requests

url = 'http://localhost:8088'
username = 'admin'
password = 'admin'

auth_res = requests.post(
    f'{url}/api/v1/security/login',
    json={'username': username, 'password': password}
)

auth_token = auth_res.json()['access_token']_

**Comment by cw1427** [comment]
![image](https://github.com/apache/superset/assets/4980410/25398085-4bf4-4357-98aa-dad09527ba16)
Try this way to login

**Comment by guruLinarc** [comment]
Facing the same issue, Even after using the above method, getting 500 Fatal error, [Discussion link](https://github.com/apache/superset/discussions/28510) please have a look.
